src/prefabs/dataContainer.js

```diff
@@ -51,10 +53,22 @@
 
     React.useEffect(() => {
       if (data) {
-        setThisPageState(prevState => ({
-          ...prevState,
-          model: data.model,
-        }));
+        switch (buttonGroupValue) {
+          case 'anotherPage':
+            setAnotherPageState(prevState => ({
+              ...prevState,
+              model: data.model,
```

Comment:
Do you still need to store the model in `state`? `data` is updating itself when the`model_id` changed